Lib/sysconfig/__main__.py

```diff
@@ -14,7 +15,6 @@
     parse_config_h,
 )
 
-
```

Comment:
Please revert this blank line removal (but I'm fine with the change on `print (`).